internal/nodes/dequeue_ingestion.go

```diff
@@ -154,6 +153,15 @@ func DequeueIngestion(key string, fullsync bool) {
 				PublishKeyToRestLayer(model_name, key, sharedQueue)
 			}
 		}
+		serviceNamespaceName := namespace + "/" + name
+		found, oldKey := objects.SharedlbLister().GetServiceToSharedVipKey(serviceNamespaceName)
+		if found {
+			// This service is found in the Shared vip LB list - this means it's a transition from Shared vip LB to clusterIP or NodePort.
+			objects.SharedlbLister().RemoveSharedVipKeyServiceMappings(serviceNamespaceName)
+			utils.AviLog.Infof("key: %s, msg: service %s transitioned from type Shared vip loadbalancer to ClusterIP or NodePort, will delete model, oldkey: %s", key, name, oldKey)
```

Comment:
Not all case we delete model for SharedVIPkey.. so we should say.. we will remove service from sharedVIP object like that

Reply:
Done